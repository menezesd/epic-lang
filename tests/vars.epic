func main() {
    first();
    x = 1;
    {
        x = 4;
        print x;
        u = 8;
    }
    print x;
    print u;

    for i in 1..3 do {
        x = x + 1;
        print x;
        u = 9;
    }
    print x;
    print u;
}

func first() {
    x = 5;
    second();
    print x;
}

func second() {
    x = 1;
    print x;
}