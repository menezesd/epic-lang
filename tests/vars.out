1
5
4
4
8
5
6
6
9
